NOTES.md

```diff
@@ -1,3 +1,143 @@
+# Release 2.2.0 (Modern Times)
+
+Released on 2015-11-XX
+
+Release highlights:
+
+* Atomic changefeeds
+* Performance & Scalability improvements
+
+Read the [blog post][2.2-release] for more details.
+
+[2.2-release]: http://rethinkdb.com/blog/2.2-release/
+
+## Compatibility ##
+
+Data files from RethinkDB versions 1.16.0 onward will be automatically migrated.
+As with any major release, back up your data files before performing the upgrade.
+
+If you're upgrading from RethinkDB 1.14.x or 1.15.x, you need to migrate your secondary
+indexes first. You can do this by following these steps:
+* install RethinkDB 2.0.5
+* update the RethinkDB Python driver (`sudo pip install 'rethinkdb<2.1.0'`)
+* run `rethinkdb index-rebuild`
+
+Afterwards, you can install RethinkDB 2.2 and start it on the existing data files.
+
+If you're upgrading directly from RethinkDB 1.13 or earlier, you will need to manually
+upgrade using `rethinkdb dump`.
+
+### API-breaking changes ###
+
+* Changefeeds on `.orderBy.limit` as well as `.get` queries previously provided
+  initial results by default. You now need to pass the `includeInitial: true` optarg to
+  `.changes` to achieve the same behavior.
+* The protocol buffer driver protocol is no longer supported. The JSON-based protocol is
+  now the only supported driver protocol. Certain older drivers (e.g. the Java drivers by
+  @dkhenry and by @npiv) no longer work with RethinkDB 2.2.0. See the [drivers][drivers]
+  list for up-to-date drivers.
+* Certain argument errors that used to throw `ReqlDriverError` exceptions now throw
+  `ReqlCompileError` exceptions. See [#4669][issue-4669] for a full list of changes.
+
+[drivers]: http://rethinkdb.com/docs/install-drivers/
+[issue-4669]: https://github.com/rethinkdb/rethinkdb/issues/4669#issue-100428248
+
+### Supported distributions ###
+
+RethinkDB 2.2.0 now comes with official packages for Ubuntu 15.10 (Wily Werewolf) and
+CentOS 7.
+
+We no longer provide packages for Ubuntu 10.04 (Lucid Lynx).
+
+## New features ##
+
+* Added full support for atomic changefeeds through the `include_initial` optarg (#3579)
+* Added a `values` command to obtain the values of an object as an array (#2945)
+* Added a `conn.server` command to identify the server for a given connection (#3934)
+* Extended `r.uuid` to accept a string and work as a hash function (#4636)
+
+## Improvements ##
+
+* Server
+ * Improved the scalability of many range queries on sharded tables (#4343)
```

Comment:
Remove "many", it makes the sentence ambiguous (many queries at one time, or many common range queries?)